### PR TITLE
add Prometheus metrics for nodes with missing topology

### PR DESCRIPTION
**Pull request by dmitsh**

### CODE REVIEW

File: pkg/providers/aws/instance_topology.go [hunk 1]
```diff
@@ -184,7 +186,7 @@ func toGraph(topology []types.InstanceTopology, cis []common.ComputeInstances) (
 				ID:   instanceID,
 			}
 		}
-		forest["cpu-nodes"] = sw
+		forest["no-topology"] = sw
```
Comment (henryh2) [inline pkg/providers/aws/instance_topology.go]: So is this "no-topology" vertex going to be a hard-coded switch for tree topology? In that case, should we placed it in the consts file in common and edit some of the tests (such as in grpc_client_test.go) to follow the same strucutre?

File: pkg/providers/oci/instance_topology.go [hunk 2]
```diff
@@ -217,7 +219,7 @@ func toGraph(bareMetalHostSummaries []*core.ComputeBareMetalHostSummary, cis []c
 				ID:   instanceID,
 			}
 		}
-		forest["cpu-nodes"] = sw
+		forest["no-topology"] = sw
```
Comment (henryh2) [inline pkg/providers/oci/instance_topology.go]: Same as previous comment with AWS nodes.

Comment (dmitsh) [reply pkg/providers/oci/instance_topology.go]: agree

Comment (dmitsh) [reply pkg/providers/oci/instance_topology.go]: done

**Review by XRFXLP — APPROVED** [review]
LGTM

**Pull request merged**